docs/manage/options_ini.rst

```diff
@@ -51,3 +39,19 @@ If you need to :ref:`profile server requests <profile_requests>` to get a more d
     
      [Server]
      PROFILE = 1
+
+
+.. _content_fallback_ini:
```

Comment:
@kollivier I'm not even sure if this is something we want to expose in our main user guide...? :thinking: